app/Models/Entry.php

```diff
@@ -558,6 +558,12 @@ public static function getContentByParsing(string $url, string $path, array $att
 			$nodes = $xpath->query(new Gt\CssXPath\Translator($path));
 			if ($nodes != false) {
 				foreach ($nodes as $node) {
+					if ($attributes['path_entries_filter'] != '') {
```

Comment:
```suggestion
					if (!empty($attributes['path_entries_filter'])) {
```

More robust if this array entry does not exist at all